### PR TITLE
Last fixes

### PR DESCRIPTION
**Pull request by eliegaboriau**
add missing translations
add check when module not initialized

### CODE REVIEW

File: app/controllers/decidim/admin/application_controller.rb [hunk 1]
```diff
@@ -56,6 +56,7 @@ def permission_scope
       end
 
       def flash_points
+        return unless current_organization.assistant
         return unless current_organization.assistant["flash"] != ""
```
Comment (Quentinchampenois) [inline app/controllers/decidim/admin/application_controller.rb:60]:
```suggestion
        return if current_organization.assistant["flash"].empty?
```

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]
Just a suggestion otherwise it seems good to me !

**Comment by paulinebessoles** [comment]
Thank you @eliegaboriau! I'm just redirected on this error when trying to access the admin panel:

Capture d’écran 2022-09-08 à 17 31 11

Could you please fix it?

File: config/locales/fr.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ fr:
   decidim:
     admin:
       assistant:
+        cannot_activate: Le module ludens c'a pas été initialisé, vous ne pouvez donc pas l'utiliser pour l'instant. Plus d'informations <a href='https://github.com/OpenSourcePolitics/decidim-ludens/blob/master/README.md'> ici </a>
```
Comment (paulinebessoles) [inline config/locales/fr.yml]:
```suggestion
        cannot_activate: L'assistant à la participation n'a pas été initialisé, vous ne pouvez donc pas l'utiliser pour l'instant. Pour l'initialiser, nous vous recommandons de consulter <a href='https://github.com/OpenSourcePolitics/decidim-ludens/blob/master/README.md'>cette notice</a>.
```

**Review by paulinebessoles — APPROVED** [review]
Works perfectly, thank you @eliegaboriau !

**Review by Quentinchampenois — APPROVED** [review]

**Pull request merged**